.circleci/config.yml

```diff
@@ -42,6 +42,12 @@ commands:
           environment:
             DATABASE_URL: 'postgres://psqlextra:psqlextra@localhost:5432/psqlextra'
 
+      - run:
+          name: Run tests psycopg3
+          command: tox -e 'py<< parameters.pyversion >>-dj{<< parameters.djversions >>}'
```

Comment:
How does this install psycopg3? I don't see any references in `tox.ini` to psycopg3.

Reply:
oh sorry, this was me trying to figure tox/circle out

Right now the djversion=42 uses psycopg3 (though It means that it does not test psycopg2)